include/deal.II/base/qprojector.h

```diff
@@ -327,10 +327,10 @@ class QProjector
      * projected onto the faces) has.
      */
     static DataSetDescriptor
-    face(const ReferenceCell &reference_cell,
-         const unsigned int   face_no,
-         const unsigned char  combined_orientation,
-         const unsigned int   n_quadrature_points);
+    face(const ReferenceCell     &reference_cell,
+         const unsigned int       face_no,
+         const types::orientation combined_orientation,
```

Comment:
At some point soon I'd also like to deprecate the three old boolean orientation functions (orientation, rotation, and flip) and simply call this the 'orientation'.